app/models/favorite.rb

```diff
@@ -0,0 +1,14 @@
+class Favorite
+  attr_reader :id, :user_id, :tweet_id, :errors
+
+  def initialize(id:, user_id:, tweet_id:, errors:)
+    @id = id
+    @user_id = user_id
+    @tweet_id = tweet_id
+    @errors = errors
```

Comment:
errorsは、引数で初期化するのではなく、Tweetクラスのようにしよう！

```suggestion
    @errors = []
```